scheduler-jobs/deploy_data_pavics_jupyter.env

```diff
@@ -0,0 +1,25 @@
+# To use in birdhouse-deploy, source this file in env.local before sourcing deploy_data_specific_image_job.env.
+# This will configure deploy_data_specific_image_job.env.
+
+#############
+## Sample way to override default configs in this file.
+#
+## Source this file providing default config values.
+#. <path to this file>
+#
+## Override cron schedule.
+#DEPLOY_DATA_JOB_SCHEDULE="3 0,3,6,9,12,15,18,21 * * *"  # UTC
+#
+## Source deploy_data_image_specific_job.env that will actually perform the job creation.
+#. <path to deploy_data_image_specific_job.env>
+#
+## End Sample
+#############
+
+DEPLOY_DATA_JOB_SCHEDULE="${AUTODEPLOY_NOTEBOOK_FREQUENCY}"
+DEPLOY_DATA_JOB_JOB_NAME="notebookdeploy-${DEPLOY_DATA_JOB_JUPYTERHUB_IMAGE_NAME}"
```

Comment:
I suggest, to make it simple, merge your `birdhouse/components/scheduler/deploy_data_specific_image_job.env ` with this one because you are rolling a custom cronjob generation code to match `deploy_data_specific_image` anyways.

I have a two-part `deploy_data_job.env` generic part and an specific override because it works for many jobs.  Yours work only for 1 specific job, might as well make it simple and have everything in this file, including the looping part so the activation in `env.local` is very simple for users.